Lib/test/_test_multiprocessing.py

```diff
@@ -6441,6 +6441,134 @@ def test_namespace(self):
         o.y = 1
         self.run_worker(self._test_namespace, o)
 
+    @classmethod
+    def _test_set_operator_symbols(cls, obj):
```

Comment:
I know that the other tests do it like that but I think it's barely readable in general. I plan to open a separate issue where we can rewrite the tests (maybe it's better to first rewrite them, backport the refactor, and use the new form).